content/03.methods.md

```diff
@@ -87,3 +87,14 @@ We annotated putative driver fusions and prioritized fusions lists with kinases,
 We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
 
 ### Clinical Data Harmonization
+
+#### Sex Determination
+
+We examined the concordance of genetic information with reported gender to identify mislabeled or contaminated samples.
```

Comment:
```suggestion
The clinical metadata provided included a reported gender.
We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.
This analysis could also reveal samples that may have been contaminated in certain circumstances.
```
@jaclyn-taroni and @jharenza I'd love another pair of eyes on this. Also, we can only get contamination in certain cases, I'd imagine. Where the sex of the contaminating sample or sample handler was different. Should we go into that here, or is it obvious?

Reply:
`We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.` The prediction done in concert, but independently, then plotted against the reported gender. For that one contaminated sample, it actually never made it to our PBTA release due to being removed for a QC issue, so it was a good check in that it fell outside of our cutoffs for being either male or female. We should note the cutoffs for M/F and I would say something like `within X range (which we should note), we deemed samples as contaminated.`